apps/nfid-frontend/src/features/transfer-modal/components/swap.tsx

```diff
@@ -194,11 +199,18 @@ export const SwapFT = ({ onClose }: ISwapFT) => {
     if (!sourceAmount || !targetAmount || !sourceUsdAmount || !targetUsdAmount)
       return
 
-    setIsSuccessOpen(true)
-
     if (!shroff) return
 
-    await shroff.validateQuote()
+    try {
+      await shroff.validateQuote()
+    } catch (e) {
+      setSlippageQuoteError((e as SlippageQuoteError).message)
+      // DISCUSS: maybe we need to set the updated quote!
```

Comment:
!

Reply:
we will discuss later.
Just a notification to keep it in mind